tests/base.go

```diff
@@ -73,78 +64,80 @@ func RemoteRun(user string, addr string, cmd string) (string, error) {
 	// Connect
 	client, err := ssh.Dial("tcp", net.JoinHostPort(addr, "22"), config)
 	if err != nil {
-		return "", err
+		return "", errors.Wrapf(err, "error starting ssh connection ")
 	}
 	// Create a session. It is one session per command.
 	session, err := client.NewSession()
 	if err != nil {
-		return "", err
+		return "", errors.Wrapf(err, "error creating new session")
 	}
 	defer session.Close()
 	var b bytes.Buffer  // import "bytes"
 	session.Stdout = &b // get output
```

Comment:
remove the comment, what we are doing here is setting the Stdout to our buffer to write to it, as it implements the io.Writer interface

Reply:
I suggest you don't do that, and use the CombinedOutput https://pkg.go.dev/golang.org/x/crypto/ssh#Session.CombinedOutput